packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -85,15 +85,51 @@ async function setupTest() {
   const addLiquidityInvite =
     await autoswapPublicFacet.makeAddLiquidityInvitation();
 
+  // Create attestationExample instance
```

Comment:
This adds a `bundleSource`, `install`, and `startInstance` to every test, right? 9 of them, it seems. How about putting the `bundleSource` and `install` in `test.before()`?

I would measure the performance impact, but I'm not sure how:

```
agoric-sdk/packages/wallet$ yarn test api/test/test-lib-wallet.js 
yarn run v1.22.17
$ exit 0 api/test/test-lib-wallet.js
Done in 0.05s.

agoric-sdk/packages/wallet$ yarn test
yarn run v1.22.17
$ exit 0
Done in 0.05s.
```

Reply:
@turadg who set up the ci for this package? git blame shows 2f0073a for these lines...

```yaml
      - name: yarn test (wallet/api)
        run: cd packages/wallet/api && yarn test
```

that commit has you as author, but it's only a formatting change.

Reply:
"this package" should be `packages/wallet/api` to run the test, which is what CI does.  `packages/wallet` is an empty stub node.  Were it enabled, it would run the tests twice.

Reply:
The tests work in current master's `cd packages/wallet/api && yarn test`.

Reply:
I just pushed a commit to optimize the tests. It was taking 13 seconds on my machine before the changes, 26 seconds after the changes, and now 6 seconds after optimizing the contract installation/instantiation

Reply:
> SyntaxError: Unexpected token '==='
I messed up checking out the branch; I used `git pull` at one point and got merge conflicts.